### ISSUE TITLE
Focus broken on tvos using clean renative-template-hello-world install

### ISSUE THREAD

**Issue by wilson208** [renative-org/renative#800]
**Describe the bug**
I am having issues getting focus to work when running on tvos devices. I am building a renative TV app where I have been primarily focussing on AndroidTV and FireTV to start with, now I am moving onto getting tvos platform working I am unable to focus on any element. 

I assumed I was doing something wrong and so I spun up a fresh install using `rnv new ...` and selecting template `renative-template-hello-world`. The install was successful however when I run the tvos app I am unable to navigate in the tvos simulator, no buttons / menu items will receive focus. These buttons are focusable when running with AndroidTV and WebOS.

**To Reproduce**

- `rnv new`
- Follow wizard and select `renative-template-hello-world` template
- rnv run -p tvos

**Expected behavior**
I expect to be able to use the remote to navigate the menu / buttons

**Actual behavior**
I am unable to navigate, and no components on screen are focused, unlike on AndroidTV or FireTV

**Desktop (please complete the following information):**

-   OS: macos Big Sur 11.6 (MacBook Pro Apple M1)
-   Node Version [e.g. v16.13.2]
-   RNV Version [e.g. 0.34.0]
-   TVOS Version: 15.2

**Comment by mihaiblaga89** [comment]
@wilson208 @Cilveti can you please try the latest version? Just tried to replicate it but it seems to work just fine

![CleanShot 2022-03-19 at 20 44 30](https://user-images.githubusercontent.com/4787898/159136120-c21a28f0-428a-449e-a10c-ccfac736078a.gif)

**Comment by wilson208** [comment]
Thanks @mihaiblaga89, I created a fresh app using hello-world template and rnv version `0.36.0-canary.4` and tvos app focus is working as expected.

To make sure I wasn't doing anything wrong first time around I did the same with a clean hello-world app with rnv version 0.35.3 and focus was broken again.

So can confirm this issue is fixed in latest canary build so going to close, thanks for your help!
